### PR TITLE
Migrate: Viewing service logs

### PR DESCRIPTION
**Pull request by sluhur**
# What changed, and why it matters

This is the migration #1072
https://github.com/aiven/devportal/issues/1072

### CODE REVIEW

**Comment by ftisiot** [comment]
Hi @sluhur, the build is failing due some missing ®, could you please check?

**Comment by sluhur** [comment]
> 

@ftisiot I've added the trademark.

File: docs/platform/howto/access-service-logs.rst [hunk 1]
```diff
@@ -0,0 +1,24 @@
+Access service logs
+====================
+
+Accessing logs from Aiven service: PostgreSQL®, Kafka®, Cassandra®, Redis®*, InfluxDB®
```
Comment (ftisiot) [inline docs/platform/howto/access-service-logs.rst:4]: What about the other services? can I access the log for M3, flink and all the missing ones?

File: docs/platform/howto/access-service-logs.rst [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+    Logs can be browsed back in time, but scrolling up several thousand lines is not very convenient.
+
+* :doc:`Aiven CLI </docs/tools/cli>` supports programmatically downloading logs with the :ref:`dedicated command <avn-service-logs>`.
+
+* `Aiven API <https://api.aiven.io/doc/#operation/ProjectGetServiceLogs>`_ endpoint is available for fetching the same information two above methods output, in case programmatic access is needed.
+
+.. Warning::
+
+    Service logs included on the normal service price are stored only for a few days. Unless you are using :doc:`logs integration </docs/integrations>` to another service, older logs are not accessible.
```
Comment (ftisiot) [inline docs/platform/howto/access-service-logs.rst:22]: what is a "normal service price"? can we be more precise?

**Review by ftisiot — CHANGES_REQUESTED** [review]
Hi @sluhur, thanks for this!
I made some modification to fit more the devportal style (also movin from Elasticsearch to Opensearch terminology). I left a couple of question where more in-dept research is needed or wording needs to be changed. Could you please have a look?

**Comment by lornajane** [comment]
Minor point from me, @sluhur, and I think we have mentioned this before. Please use meaningful titles and descriptions for your pull requests, as explained in the CONTRIBUTING guide, we have a very large number of PRs to track and review, and yours is less likely to be handled in a timely manner if the information is incomplete. I edited the title, but the description is not correctly linked to the issue, could you check it please?

**Comment by sluhur** [comment]
> Minor point from me, @sluhur, and I think we have mentioned this before. Please use meaningful titles and descriptions for your pull requests, as explained in the CONTRIBUTING guide, we have a very large number of PRs to track and review, and yours is less likely to be handled in a timely manner if the information is incomplete. I edited the title, but the description is not correctly linked to the issue, could you check it please?

Apologize for this. I'm pretty new with this and still learning. I've linked to the issue now. Let me know if there's anything that I should be doing.

File: docs/platform/howto/access-service-logs.rst (hunk 2 above)
Comment (sluhur) [reply docs/platform/howto/access-service-logs.rst]: I'm removing the normal service price as it applies to all services actually.

File: docs/platform/howto/access-service-logs.rst (hunk 1 above)
Comment (sluhur) [reply docs/platform/howto/access-service-logs.rst]: I'm removing the line as well as all services will have logs.

**Review by ftisiot — APPROVED** [review]
LGTM

**Comment by ftisiot** [comment]
Before merging, if @sluhur you could fix the description, to add the `Fix #isssue_number` in it, so once we merge this also the original issue will be closed

**Comment by ftisiot** [comment]
Hi @sluhur, I don't quite understand the last changes? I believe the title and the filename was changed to `access` from `viewing` in one of the reviews, why going back?

**Comment by sluhur** [comment]
> Hi @sluhur, I don't quite understand the last changes? I believe the title and the filename was changed to `access` from `viewing` in one of the reviews, why going back?

Sorry @ftisiot , I miss understood with your question about changing the description. I've changed it back to access service logs.

**Review by ftisiot — APPROVED** [review]
LGTM

**Pull request merged**